voyager-publish/bin/voyager-publish.js

```diff
@@ -48,11 +49,12 @@ console.log('Unlocking previously locked measurements')
 // FIXME
 console.log('Skipping unlocking measurements')
 // await client.query(
-//   'UPDATE measurements SET locked_by_pid = NULL WHERE locked_by_pid IS NOT NULL'
+//   'UPDATE measurements SET lock = NULL WHERE lock IS NOT NULL'
 // )
 console.log('Unlocked measurements')
 await Promise.all(new Array(CONCURRENCY).fill().map(async () => {
   while (true) {
+    const lock = randomUUID()
```

Comment:
@bajtos I went for UUIDs since they're slightly simpler to implement than a counter (no state to keep)